js/templates/pageNav.html

```diff
@@ -16,30 +16,30 @@
       </div>
       <div>
         <div class="flexVCent">
-          <a class="iconBtn ion-chevron-left js-navBack"></a>
+          <a class="iconBtn margLSm ion-chevron-left js-navBack"></a>
           <a class="iconBtn ion-chevron-right js-navFwd"></a>
-          <a class="iconBtn ion-refresh js-navReload"></a>
+          <a class="iconBtn ion-refresh js-navReload margRSm"></a>
```

Comment:
The extra spacing on the left and right of the buttons seems pretty specific, I think it would make sense to add a class in the page nav style sheet, and put it on the parent (flexVCent) as left and right padding.